docs/standard/serialization/how-to-qualify-xml-element-and-xml-attribute-names.md

```diff
@@ -36,6 +36,7 @@ By creating an instance of `XmlSerializerNamespaces` and adding the namespace pa
 
 The following example creates an `XmlSerializerNamespaces`, and adds two prefix and namespace pairs to the object. The code creates an `XmlSerializer` that is used to serialize an instance of the `Books` class. The code calls the `Serialize` method with the `XmlSerializerNamespaces`, allowing the XML to contain prefixed namespaces.
 
+<!-- TODO: THE FOLLOWING VB SNIPPET ISN'T CORRECT!! -->
```

Comment:
**Please, pay attention to this..**

Reply:
Thanks @Youssef1313. Can you please file an issue for this?

Reply:
@mairaw, Done. #15809